.gitignore

```diff
@@ -1,3 +1,5 @@
+# The 5 approved on boarding samples that need wwwroot can add a local .gitignore that allows wwwroot.
```

Comment:
The change in https://github.com/aspnet/Docs/pull/4532 removed ignore lines for wwwroot/lib, not all of wwwroot.  Are you sure it won't create problems to ignore all of wwwroot?  Don't some projects put project-specific files in there on occasion?